src/app/devtools/networklogger/qgsnetworklogger.h

```diff
@@ -76,6 +76,8 @@ class QgsNetworkLogger : public QAbstractItemModel
     QList< QAction * > actions( const QModelIndex &index, QObject *parent );
 
 
+    using QAbstractItemModel::removeRows;
+
     /**
```

Comment:
This one could just be renamed -- it's not part of public api

Reply:
any suggestion?

Reply:
removeRequestRows?

Reply:
done